src/components/MessageList/MessageNotification.js

```diff
@@ -14,7 +14,7 @@ const Container = styled.TouchableOpacity`
   height: 27px;
   width: 112px;
   z-index: 10;
-  margin-bottom: 0;
+  margin-bottom: 0px;
```

Comment:
isn't this normally 0 by default? so we wouldn't need to specific a margin-bottom?